app/views/pages/search.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, t("search_view.title")) %>
+<% provide(:base_title, t("base.title")) %>
+<div class="col-lg-9">
+  <h1><%= t "search_view.h1"%></h1>
+  <hr>
+  <div class="row">
+    <% @locations.each do |location| %>
+      <%= render @locations %>
```

Comment:
Đã render kiểu collection partial rồi sao còn đặt trong vòng for each?

Reply:
dạ em sửa lại rồi ạ , chổ này em nắm chưa kỉ ạ